CAAppKitBridge/Demo/main.m

```diff
@@ -0,0 +1,90 @@
+/* Demo/main.m
+
+   Copyright (C) 2018 Free Software Foundation, Inc.
+
+   Author: Stjepan Brkic <stjepanbrkicc@gmail.com>
+   Date: June 2018
+
+   This file is part of QuartzCore.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; see the file COPYING.LIB.
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110-1301, USA.
+*/
+
+#import "../Source/GSCAData.h"
+#import "../Source/NSView+CAMethods.h"
+#import <Foundation/Foundation.h>
+
+@interface DemoController : NSObject
+{
+}
+@end
+
+@implementation DemoController
+- (void) applicationDidFinishLaunching: (id) t
+{
+  NSView * view = [[NSView alloc]init];
+  NSView * view2 = [[NSView alloc]init];
+  NSView * view3 = [[NSView alloc]init];
+  NSView * view3_5 = [[NSView alloc]init];
+  NSView * view4 = [[NSView alloc]init];
+
+  NSLog(@"%p %p %p %p %p",view,view2,view3,view3_5,view4);
+
+  [view addSubview:view2];
```

Comment:
Space after colon. (GNU code style.)

Reply:
You already mentioned this

Reply:
As discussed on the call, I have enumerated this problem in as many places as possible so you can treat this almost as a checklist. Think of this as a dumb, manual linter.

Of course, this should be resolved even where I have not listed the problem.